.gitignore

```diff
@@ -1,126 +1,15 @@
-# Created by https://www.gitignore.io/api/flask
-# Edit at https://www.gitignore.io/?templates=flask
-
-### Flask ###
-instance/*
-!instance/.gitignore
-.webassets-cache
-
-### Flask.Python Stack ###
-# Byte-compiled / optimized / DLL files
+#python cache 
 __pycache__/
-*.py[cod]
-*$py.class
-
-# C extensions
-*.so
-
-# Distribution / packaging
-.Python
-build/
-develop-eggs/
-dist/
-downloads/
-eggs/
-.eggs/
-lib/
-lib64/
-parts/
-sdist/
-var/
-wheels/
-pip-wheel-metadata/
-share/python-wheels/
-*.egg-info/
```

Comment:
I believe doing `poetry install` will create an .egg-info folder, unless you run it with --no-root

Reply:
Thanks!